hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/HRegion.java

```diff
@@ -8730,12 +8868,22 @@ public void startRegionOperation() throws IOException {
 
   @Override
   public void startRegionOperation(Operation op) throws IOException {
+    boolean isInterruptableOp = false;
     switch (op) {
-      case GET:  // read operations
+      case GET:  // interruptible read operations
       case SCAN:
+        isInterruptableOp = true;
```

Comment:
Only one q left, read operations are interruptible, but i couldn't find new interruption handling in read path. Do we just leave it as it is? (just for confirmation)

Reply:
There are some checkInterrupts placed inside the scanner inner loops.